.github/README.md

```diff
@@ -47,98 +50,98 @@ log: info
 #         e. g. with the slug 'ethereum' and the indexer EthBlobs the resulting url will be: "/ethereum/beacon/blob_sidecars?..."
 # =============
 pools:
-    - chainid: kaon-1
-      indexer: EthBlobs
-      poolid: 21
-      slug: ethereum
-    - chainid: korellia-2
-      indexer: Height
-      poolid: 105
-      slug: linea
+  - chainid: kaon-1
+    indexer: EthBlobs
+    poolid: 21
+    slug: ethereum
+  - chainid: korellia-2
+    indexer: Height
+    poolid: 105
+    slug: linea
```

Comment:
Please add `proofAttached`

Reply:
I feel like `proofAttached` should be true by default. Maybe we should rename this flag to `excludeProof` and make it optional.